src/model/error.messages.ts

```diff
@@ -1,5 +1,6 @@
 export const ERROR_SUMMARY_TITLE: string = "There is a problem";
-
 export const INVALID_COMPANY_NUMBER: string = "Invalid company number";
 export const NO_COMPANY_NUMBER_SUPPLIED: string = "No company number supplied";
 export const COMPANY_NUMBER_TOO_LONG: string = "Company number too long";
+export const NO_COMPANY_NUMBER_IN_SESSION: string = "No company number found in session";
+export const COMPANY_NUMBER_NOT_FOUND: string = "Company number not found";
```

Comment:
See previous comment - should this be 'Company not found'?

Reply:
Yes it should.